### DISCUSSION TITLE
Does MVT-iOS retrieve data of remote access to phone (details)?

### DISCUSSION THREAD

**Discussion by citycritter** [mvt-project/mvt#420] [Q&A] [answered]
I think my coworkers found a way to watch me through my phone. I don't think Pegasus was used, but would MVT have a way to view camera, screen, and microphone access history from remote viewers? Including dates and time? Other suggestions would be appreciated.

**Comment by Te-k** [comment]
Hi,
What kind of phone do you have?

**Reply by citycritter** [reply]
I have an iPhone 14 Pro

**Reply by Te-k** [reply, marked as answer]
Remote viewers do no exist in iPhone afaik. Considering the default security in iPhones, it is quite unlikely that your coworker would have access to technology to compromise it, the most likely scenario imho would be another form of surveillance or access to your icloud account if they managed to get access to its password.
That said you can find some data by looking at the iPhone backup, for instance the [locationd](https://docs.mvt.re/en/latest/ios/records/#locationd_clientsjson) artifact list all apps that requested access to geolocation

**Reply by citycritter** [reply]
Thank you, I feel like I'm headed in the right direction. I'll look through suggested and update. xx
